ops/solana/solana.go

```diff
@@ -265,23 +267,43 @@ func (d *Deployer) DeployOCR() error {
 	err = d.gauntlet.ExecCommand(
 		"store:set_writer",
 		d.gauntlet.Flag("network", d.network),
-		d.gauntlet.Flag("ocrState", d.Account[OCRFeed]),
 		d.gauntlet.Flag("input", string(jsonInput)),
+		d.Account[OCRFeed],
 	)
 
 	if err != nil {
 		return errors.Wrap(err, "setting writer on store failed")
 	}
+
+	fmt.Println("Step 8: Transfer feed ownership to store")
+	if err = d.gauntlet.ExecCommand(
+		"store:transfer_feed_ownership",
+		d.gauntlet.Flag("network", d.network),
+		d.gauntlet.Flag("state", d.Account[OCRTransmissions]),
+		d.gauntlet.Flag("to", d.Account[StoreAccount]),
+	); err != nil {
+		return errors.Wrap(err, "failed to transfer feed ownership")
+	}
+
+	if err = d.gauntlet.ExecCommand(
+		"store:accept_feed_ownership",
+		d.gauntlet.Flag("network", d.network),
+		d.gauntlet.Flag("state", d.Account[OCRTransmissions]),
+		d.gauntlet.Flag("to", d.Account[StoreAccount]),
+	); err != nil {
+		return errors.Wrap(err, "failed to accept feed ownership")
+	}
+
 	return nil
 }
 
 func (d Deployer) TransferLINK() error {
 	err := d.gauntlet.ExecCommand(
-		"token:transfer",
+		"ocr2:fund",
 		d.gauntlet.Flag("network", d.network),
-		d.gauntlet.Flag("to", d.Account[OCRFeed]),
 		d.gauntlet.Flag("amount", "10000"),
-		d.Account[LINK],
+		d.gauntlet.Flag("link", d.Account[LINK]),
+		d.Account[OCRFeed],
 	)
 	if err != nil {
 		return errors.Wrap(err, "LINK transfer failed")
```

Comment:
```suggestion
		return errors.Wrap(err, "'ocr2:fund' call failed")
```

If this is surfaced through an original error, maybe even is unnecessary to wrap.